packages/app/src/debug/DebugPageHeader.cy.tsx

```diff
@@ -20,12 +20,13 @@ describe('<DebugPageHeader />', {
             result.commitInfo.summary = 'Adding a hover state to the button component'
             result.commitInfo.branch = 'feature/DESIGN-183'
             result.commitInfo.authorName = 'cypressDTest'
+            result.commitInfo.sha = 'b5e6fde'
           }
         }
       },
       render: (gqlVal) => {
         return (
-          <DebugPageHeader gql={gqlVal} runNumber={468} commitsAhead='You are 2 commits ahead' commitHash='b5e6fde'/>
+          <DebugPageHeader gql={gqlVal} commitsAhead={2}/>
```

Comment:
Can you be commits behind? Eg, you look at a test result but you haven't done `git pull` in a while. In which case this prop might be `commitDelta` or something more representative of this state.

Reply:
The use cases for this feature do not cover trying to look forward in commits that you have not pulled into your current local branch.  The scenario this is covering is only that the query to the cloud can only find a "relevant run" to your local environment that is earlier than commit you are currently on.  The normal case for this would be that you made some local changes, committed them locally, and either have not pushed them or have and CI has not run on them.  Then you would be `X` commits ahead of the "relevant run" you are currently looking at on the "Debug" page.  There are no curent plans for this feature to show you a run that is on a commit ahead of what you have locally.  There is work in a future milestone to have a different message in a banner that tells the developer that there are newer runs that they could see if they do `git pull`.